### PR TITLE
WIP provisioning

### PR DESCRIPTION
**Pull request by kdeloach**

### CODE REVIEW

**Comment by mmcfarland** [comment]
I was picturing sharing our existing workers to also execute the RWD tasks.  Is this "WIP" to just get the code executing, or working towards setting up a new machine type?

**Comment by kdeloach** [comment]
This is just a sandbox to figure out the provisioning steps required. However, it's also useful to be able to work on this module in isolation.

After I get everything working I'm planning to convert these steps to ansible roles and merge back into the main MMW project.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "precise32"
```
Comment (hectcastro) [inline Vagrantfile]: You may be operating on 12.04 here. MMW is on 14.04, or `ubuntu/trusty64`.

File: bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+sudo apt-get update
+
+# See: https://github.com/Toblerity/Fiona/issues/74
+sudo apt-get install -y python-software-properties
+sudo add-apt-repository ppa:ubuntugis/ppa
```
Comment (hectcastro) [inline bootstrap.sh]: This is for a more recent version of GDAL? Does needing this change if we are on 14.04?

File: bootstrap.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+sudo apt-get update
+
+sudo apt-get install -y g++
+sudo apt-get install -y gfortran
```
Comment (hectcastro) [inline bootstrap.sh]: Is this necessary for R? Do we have to execute R as part of a job workflow?

File: requirements.txt [hunk 4]
```diff
@@ -1 +1,23 @@
...
+pyshp==1.2.3
+python-apt===0.8.3ubuntu7
+python-dateutil==2.4.2
+python-debian===0.1.21ubuntu1
+pytz==2015.7
+scipy==0.16.1
+Shapely==1.2.19
+six==1.10.0
+ufw==0.31.1.post1
+unattended-upgrades==0.1
```
Comment (hectcastro) [inline requirements.txt]: Where are these coming from? Unable to find them on PyPi.

File: bootstrap.sh (hunk 3 above)
Comment (kdeloach) [inline bootstrap.sh]: One of the python dependencies requires this to compile source code.

File: Vagrantfile (hunk 1 above)
Comment (kdeloach) [inline Vagrantfile]: Thanks, will update.

File: requirements.txt (hunk 4 above)
Comment (kdeloach) [inline requirements.txt]: Good question, I don't know. This was the output from `pip freeze` after I manually installed dependencies.

File: bootstrap.sh (hunk 2 above)
Comment (kdeloach) [inline bootstrap.sh]: I think you're right. Will try removing this now.

File: requirements.txt (hunk 4 above)
Comment (mmcfarland) [inline requirements.txt]: According to the author, only
```
sudo pip install fiona 
sudo pip install numpy
sudo pip install pandas
sudo pip install shapely
sudo apt-get install libblas-dev liblapack-dev libatlas-base-dev gfortran
sudo pip install scipy
sudo pip install pyshp
```
should be required for the python dependencies.

Comment (kdeloach) [inline requirements.txt]: I will remove auxiliary dependencies.

**Comment by kdeloach** [comment]
Simplified things a bit. However, I'm having trouble with Virtualbox which I'll have to resolve before I continue working on this.

**Pull request closed without merging**